index.html

```diff
@@ -420,6 +473,9 @@ <h3>
           async function doPaymentRequest() {
             try {
               const request = new PaymentRequest(methodData, details, options);
+              // See below for a detailed example of handling these events
```

Comment:
✅

Reply:
Hi @marcoscaceres - I'm curious: What are these checkmarks indicating? Do you need our special attention on these lines?

Are these the resolutions to previous review comments? GitHub does not seem to show any other comments for these lines. (If there is a way to show that in GitHub, then I can't find it 😆 )

Reply:
Sorry, the check boxes are just Ian and I going through the document line by line to see if everything was brought back properly. 

It's just so we know where we were.